module/VuFind/src/VuFind/Config/Upgrade.php

```diff
@@ -709,6 +709,13 @@ protected function upgradeConfig()
             unset($newConfig['Syndetics']['url']);
         }
 
+        // Convert spellchecker 'simple' option
+        if (isset($newConfig['Spelling']['simple']) && !isset($newConfig['Spelling']['dictionaries'])) {
+            $newConfig['Spelling']['dictionaries'] = $newConfig['Spelling']['simple']
+                ? ['basicSpell'] : ['default', 'basicSpell'];
+            unset($newConfig['Spelling']['simple']);
+        }
+
```

Comment:
I added a draft rule to upgradeConfig; have not used that before.  It seems to work ok using the upgrade utility but I am not not sure I understand it completely.

Reply:
Looks good to me!